utils/__tests__/auth_jwt.test.ts

```diff
@@ -0,0 +1,108 @@
+import * as jwt from "jsonwebtoken";
+
+import * as E from "fp-ts/Either";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+import { AuthJWT, getGenerateAuthJWT, getValidateAuthJWT } from "../auth_jwt";
+
+import { JWTConfig } from "../config";
+
+import { aPrimaryKey } from "../../__mocks__/keys";
+import { getGenerateJWT } from "../jwt_with_key_rotation";
+import { Second } from "@pagopa/ts-commons/lib/units";
+import { pipe } from "fp-ts/lib/function";
+
+const issuer = "test-issuer" as NonEmptyString;
+const standardJWTTTL = 900 as Second;
+
+const aPayload = {
+  assertionRef: "sha256-p1NY7sl1d4lGvcTyYS535aZR_iJCleEIHFRE2lCHt-c",
+  operationId: "anOperationId"
+} as AuthJWT;
+
+const aConfigWithPrimaryKey = pipe(
+  JWTConfig.decode({
+    ISSUER: issuer,
+    PRIMARY_PRIVATE_KEY: aPrimaryKey.privateKey,
+    PRIMARY_PUBLIC_KEY: aPrimaryKey.publicKey
+  }),
+  E.getOrElseW(_ => {
+    throw Error("Cannot decode IConfig " + JSON.stringify(_));
+  })
+);
+
+describe("getGenerateJWT", () => {
+  it("should generate a valid AuthJWT", async () => {
+    const generateJWT = getGenerateAuthJWT(aConfigWithPrimaryKey);
+
+    const res = await generateJWT(aPayload)();
+
+    console.log(res);
```

Comment:
Do we need this?

Reply:
Good catch! Removed in [3195e23](https://github.com/pagopa/io-functions-lollipop/pull/14/commits/3195e237f97d2bc6a9790be6394843bca79bdae4)